dev/com.ibm.ws.microprofile.reactive.streams.operators_fat/fat/src/com/ibm/ws/microprofile/reactive/streams/test/suite/FATSuite.java

```diff
@@ -30,30 +31,38 @@
                 ReactiveConcurrentWorkTest.class
 })
 public class FATSuite {
-    public static final String MPRS10_ID = MicroProfileActions.MP21_ID + "_" + "MPRS10";
-    public static final String MPRS30_MP60_ID = MicroProfileActions.MP60_ID + "_" + "MPRS30";
+    public static final String MP_REACTIVE_STREAMS_10 = "mpReactiveStreams-1.0";
+    public static final String MP_REACTIVE_STREAMS_30 = "mpReactiveStreams-3.0";
 
-    public static final FeatureSet MPRS10 = MicroProfileActions.MP21.addFeature("mpReactiveStreams-1.0").build(MPRS10_ID);
-    public static final FeatureSet MPRS30_MP60 = MicroProfileActions.MP60.addFeature("mpReactiveStreams-3.0").build(MPRS30_MP60_ID);
+    public static final String MP21_RSO10_ID = MicroProfileActions.MP21_ID + "_" + "mpRSO10";
+    public static final String MP50_RSO30_ID = MicroProfileActions.MP50_ID + "_" + "mpRSO30";
+    public static final String MP60_RSO30_ID = MicroProfileActions.MP60_ID + "_" + "mpRSO30";
+    public static final String MP61_RSO30_ID = MicroProfileActions.MP61_ID + "_" + "mpRSO30";
 
-    public static final Set<FeatureSet> ALL;
-    static {
-        ALL = new HashSet<>(MicroProfileActions.ALL);
-        ALL.add(MPRS10);
-        ALL.add(MPRS30_MP60);
+    public static final FeatureSet MP21_RSO10 = MicroProfileActions.MP21.addFeature(MP_REACTIVE_STREAMS_10).build(MP21_RSO10_ID);
+    //MP50 runs on Java 8 but RSO30 will only run on Java11 or higher
+    public static final FeatureSet MP50_RSO30 = MicroProfileActions.MP50.addFeature(MP_REACTIVE_STREAMS_30).setMinJavaLevel(SEVersion.JAVA11).build(MP50_RSO30_ID);
+    public static final FeatureSet MP60_RSO30 = MicroProfileActions.MP60.addFeature(MP_REACTIVE_STREAMS_30).build(MP60_RSO30_ID);
+    public static final FeatureSet MP61_RSO30 = MicroProfileActions.MP61.addFeature(MP_REACTIVE_STREAMS_30).build(MP61_RSO30_ID);
+
+    //All MicroProfile ReactiveMessaging FeatureSets - must be descending order
+    private static final FeatureSet[] ALL_RSO_SETS_ARRAY = { MP61_RSO30, MP60_RSO30, MP50_RSO30, MP21_RSO10 };
+    private static final List<FeatureSet> ALL = Arrays.asList(ALL_RSO_SETS_ARRAY);
+
+    public static RepeatTests repeatDefault(String serverName) {
+        return repeat(serverName, TestMode.LITE, FATSuite.MP61_RSO30, FATSuite.MP21_RSO10, FATSuite.MP50_RSO30, FATSuite.MP60_RSO30);
```

Comment:
So we want to run all of them in lite mode by default?

Also why isn't the repeat order in descending order like in other places?

Reply:
I will review the LITE setting there. The order of the "other" feature sets should not matter.